src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString
+    .split(';')
+    .map((line) => line.trim())
+    .filter((line) => line !== '')
+    .forEach((line) => {
+      const [key, value] = line.split(':');
```

Comment:
The current implementation splits each line by the first colon found. This means if a value contains a colon, it will not be handled correctly. However, handling such cases is not required according to the additional prompt.